src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/FileCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret file")
 public class FileCredentials extends BaseStandardCredentials {
 
-    public Control file = control("file");
+    public Control file = control(by.name("file"));
```

Comment:
The name attribute is not guaranteed to be unique on the page, though. Not sure it is a problem in this case.